addon/active-model-adapter.ts

```diff
@@ -140,22 +135,19 @@ const ActiveModelAdapter = RESTAdapter.extend({
 
     For more information on 422 HTTP Error code see 11.2 WebDAV RFC 4918
     https://tools.ietf.org/html/rfc4918#section-11.2
```

Comment:
We should probably keep the JSDoc signatures

Reply:
Done